Turkish Translation

realbaycan opened the pull request:
296 words translated.

realbaycan committed:
Turkish Translation

296 words translated.

realbaycan commented:
Çevirimi kontrol edebilir misiniz?

jnavila commented on book/02-git-basics/sections/aliases.asc:
```diff
@@ -1,70 +1,70 @@
+Aşağıda, kurmak isteyebileceğiniz birkaç örnek verilmiştir:
 
-[source,console]
+[kaynak, konsol]
```
Please don't translate the lines in the form `[...]`. These are asciidoc formatting commands. Please read about asciidoc to better understand what needs to be translated.

http://asciidoctor.org/docs/user-manual/

realbaycan committed:
Turkish Translation

Düzeltmemi yaptım tekrar kontrol eder misiniz? :)

realbaycan commented:
I made my corrections. :)

realbaycan committed:
Turkish Translate

296 words translated.

realbaycan committed:
Update aliases.asc

jnavila commented on book/02-git-basics/sections/aliases.asc:
```diff
@@ -1,5 +1,5 @@
 [[_git_aliases]]
-=== Git Takma Adları
+=== Git Aliases
```
I think you didn't understand. This part is a title and it must be translated.

OTOH, you mustn't translate the lines `[source,console]`

realbaycan replied:
sorry :) I'm fixing now

realbaycan committed:
Turkish Translation


I have edited. :)

realbaycan commented:
Thank you very much for your help. :)

realbaycan commented:
I am very be pleased if you checked other my translations.

jnavila commented:
I can only check that the translation keeps the asciidoc formatting, not the good meaning of it.

realbaycan commented:
Okey, very thanks for help

realbaycan commented:
Can I ask you something? @jnavila

realbaycan commented:
I want to continue translating but how to find the translator of the approver.
Can you help me?

jnavila commented:
I'm not sure of what I can do about this. It seems @gokaygurcan doesn't have much time to spend on managing this translation, although he seems to have the needed technical knowledge about asciidoctor and Git.

On the other hand, you are clearly wanting to help but, from your first contribution, I feel that you need guidance. Please, don't take it badly, but I don't feel also that you have an idea of what it entails to translate a book. It's not just the matter of taking a part of text and translating it (for instance by translating word to word the `TRANSLATION_NOTES.md` and `README.md` files). A book is a single piece of work that must be understood entirely before starting to translate. 

The subject of this book is Git, a computer tool that you don't seem to master either, and I don't understand a single word of Turkish. How can I have confidence that your work is not just gibberish to someone knowing a little of the subject? The thing is that you're quite alone on this task as of now, with nobody to proof-check your work.

I'm really sorry about this situation.

gokaygurcan commented:
Hi again @realbaycan and @jnavila,

I tried to explain myself in the previous PR but let's say it again: as far as I understand, @realbaycan doesn't know anything about Git and its terms/commands. This is one thing to check on my side all the time that no one tries to translate Git commands. On the other hand, since they are doing this to [earn money/Steem/token/whatever](https://utopian.io/project/progit/progit2-tr/github/29938333/all), I can't guarantee that I'll be available all the time for them to check their translations and help them to get money out of this. This is obviously not my full-time job, I'm checking and approving the translations when I have time. If this means that I'm not sufficient to help people earn money, let me be. Otherwise, I suggest you to wait till everything is read and approved. 

Thanks again for your concerns, I'll have some time later today, probably all these open PR's will be proofread and be hopefully merged. 

G.

jnavila commented:
@gokaygurcan,

In my previous reply, I didn't mean to be rude with you. I perfectly understand that this is an activity you have in your free-time (so I do), and as such, one cannot expect you to reply to all the requests at once. FYI, I'm in the process ot setting up automatic compilation of the translated books to PDFs and epub, and have them downloadable at git-scm.com. This setup adds some basic tests about the structure of the book (chapters, internal links). That could help you by unloading some basic automatic tests.

@realbaycan,

@gokaygurcan and I both agree on your actual ability to translate the book. So please, do yourself a favor: read the book, install Git and use it, then go read about asciidoc. It's always difficult to provide a first contribution to a project, but to be accepted more rapidly, it is almost always better to not get on maintainer's nerves and not give them more work than if they did it by themselves.

realbaycan committed:
Turkish Translation

800 words have been translated into Turkish.
my research on git:
Basit Git Rehberi -> http://rogerdudler.github.io/git-guide/index.tr.html
Türkçe Git 101 -> https://www.gitbook.com/book/aliozgur/git101/
Code School Try Git -> https://try.github.io/levels/1/challenges/1
Pro Git -> https://progit2.s3.amazonaws.com/en/2015-04-20-aa3b6/progit-en.452.pdf
Git Püf Noktaları -> http://vigo.github.io/git-tips/

gokaygurcan commented:
Thanks for the additions, I found lots of mistakes in the new strings, and will address them asap. And it's good to see that you're also learning Git ;)

realbaycan committed:
Turkish Translation

800 words were translated into Turkish.
my research on the git:
Basit Git Rehberi -> http://rogerdudler.github.io/git-guide/index.tr.html
Türkçe Git 101 -> https://www.gitbook.com/book/aliozgur/git101/
Code School Try Git -> https://try.github.io/levels/1/challenges/1
Pro Git -> https://progit2.s3.amazonaws.com/en/2015-04-20-aa3b6/progit-en.452.pdf
Git Püf Noktaları -> http://vigo.github.io/git-tips/

realbaycan commented:
Thanks for your help.:)
